packages/ember-routing/lib/services/router.js

```diff
@@ -62,8 +62,24 @@ const RouterService = Service.extend({
        attempted transition
      @public
    */
-  replaceWith() {
+  replaceWith(/* routeNameOrUrl, ...models, options */) {
     return this.router.replaceWith(...arguments);
+  },
+
+  /**
+     Generate a URL based on the supplied route name.
+
+     @method urlFor
+     @param {String} routeName the name of the route
+     @param {...Object} models the model(s) or identifier(s) to be used while
+       transitioning to the route.
+     @param {Object} [options] optional hash with a queryParams property
```

Comment:
HAHA! Derp.  I just was focusing on the code snippet:

```js
urlFor(routeName, ...models, queryParams)
```

Which is actually incorrect, it should have said:

```js
urlFor(routeName, ...models, { queryParams })
```

Or:

```js
urlFor(routeName, ...models, options)
```

(based on the prose just below)

https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md#new-method-url-generation